### DISCUSSION TITLE
Should I give up?

### DISCUSSION THREAD

**Discussion by MarkDMatzke** [lshqqytiger/stable-diffusion-webui-directml#169] [Q&A]
<..< Well I wanted to try to see if i could load image with Directml, But sadly I don't even know if Directml  is downloaded on mine Laptop.
These are Spec (pretty low)
Processor	AMD Ryzen 7 3700U with Radeon Vega Mobile Gfx     2.30 GHz
Installed RAM	8.00 GB (7.45 GB usable)
System type	64-bit operating system, x64-based processor

This is mine webui-user
{@echo off

set PYTHON=
set GIT=
set VENV_DIR=
set PYTORCH_CUDA_ALLOC_CONF=garbage_collection_threshold:0.6,max_split_size_mb:128
set COMMANDLINE_ARGS=--lowvram --always-batch-cond-uncond --precision full --no-half --opt-split-attention --opt-sub-quad-attention --sub-quad-q-chunk-size 512 --sub-quad-kv-chunk-size 512 --sub-quad-chunk-threshold 80 --disable-nan-check --use-cpu interrogate gfpgan codeformer --upcast-sampling --api --listen --autolaunch
set SAFETENSORS_FAST_GPU=1
call webui.bat}

And if it dose work with the GPU error it take over 10 mins for one picture sometimes it even crashed as the loading hits around 98% 
Arg! the agony  of so closed and waiting so long just to see that over and over again. 

IS there something I am missing or Just takes that long to load?

**Comment by MarkDMatzke** [comment]
Yeah >...> I gave up that end of that adventure even gotten crushed a few times >..> trouble for loading an image *sigh* >..> I found out from catbird about this whole AI adventure ( I didn't wanted to download ) Stupid me it's take 24gb of RAM just for the text modules... Why did I even think that I had chance at making images... Just like mine life another failure... Laptop here is just to weak >..> <..< sigh

**Comment by verywetfire** [comment]
> Yeah >...> I gave up that end of that adventure even gotten crushed a few times >..> trouble for loading an image _sigh_ >..> I found out from catbird about this whole AI adventure ( I didn't wanted to download ) Stupid me it's take 24gb of RAM just for the text modules... Why did I even think that I had chance at making images... Just like mine life another failure... Laptop here is just to weak >..> <..< sigh

Those 24GB text modules are for running services, not local operations as far as I understand those (I don't really, but my friend did run some text AI locally on 8 or 12 GB RAM iirc). 

If you still have your SD setup: you should've probably use --lowvram and -lowram as well since you're on very low end when it comes to SD. Also I'd rather use smaller chunk sizes like 256 or 128.

**Comment by MarkDMatzke** [comment]
Should had guess about the lowram .... >..> sadly it been to long since I was able to used the computer to anything like this before. yeah i had done this already still take about 7 min for 250 the ETA was about an hours i did take a nap HA but that was with 4 on them trying to load and some of them didn't go into details, but what is new on the computer right. >..> Gee talk about dial-up internet >..> Woah when the computer was better the internet and now it the opsited where the internet better the computer >..>

**Comment by MarkDMatzke** [comment]
did these files added anything on the Registry editor?
